ShopperInsights/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation deps.coreKtx
 
     api project(':BraintreeCore')
-
+    implementation 'com.google.code.gson:gson:2.8.9'
```

Comment:
I believe we don't want to add a 3rd party dependency. Let's check if there's an equivalent to @SerializedName that comes out of the box.

Reply:
@sshropshire wondering how we've handled property names without importing gson. Do we just use snake case where required if that's what the API expects?

Reply:
Yes we just use string values for keys (see [PayPaLineItem example](https://github.com/braintree/braintree_android/blob/main/PayPal/src/main/java/com/braintreepayments/api/PayPalLineItem.java)). 

+1 to not including third party dependencies here

Reply:
removed Gson, adding custom fromJson method